test/controller.go

```diff
@@ -70,9 +73,10 @@ type Data struct {
 
 // Clients holds references to clients which are useful for reconciler tests.
 type Clients struct {
-	Pipeline *fakepipelineclientset.Clientset
-	Resource *fakeresourceclientset.Clientset
-	Kube     *fakekubeclientset.Clientset
+	Pipeline    *fakepipelineclientset.Clientset
+	Resource    *fakeresourceclientset.Clientset
+	Kube        *fakekubeclientset.Clientset
+	CloudEvents cloudeventclient.CEClient
```

Comment:
I knew there was a better place to store this client :D only at the time I had no idea :P